src/Validation/Constraints/Parallel.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/* Copyright (c) 2017 Stefan Hecken <stefan.hecken@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\Validation\Constraints;
+use ILIAS\Validation\Constraint;
+use ILIAS\Data\Factory;
+use ILIAS\Data\Result;
+
+class Parallel extends Custom implements Constraint {
```

Comment:
If you derive this from Custom use the capabilities from Custom i.e. the constructor.